config/locales/idv/en.yml

```diff
@@ -72,6 +72,8 @@ en:
         internal_error: There was an internal error processing your request. Please try again.
         link: please contact us
         text_html: Please try again. If you keep getting these errors, %{link}.
+        try_again: We are having technical difficulties at the moment.  Try searching
+          for a Post Office again. If this issue continues, come back later.
       phone:
```

Comment:
I did not edit the internal_error message because it is being used in 5 other areas of the application.  We only need to update this one instance. I added a new variable and updated the `in-person-location-post-office-search-step.tsx` file to use the new message

Reply:
Since "Post Office" is pretty specific to in-person proofing and not IdV broadly, could we include a qualifier in the string key? Maybe something like `idv.failure.exceptions.post_office_search_error` or `idv.failure.exceptions.in_person.post_office_search_error`?

Reply:
Thanks for the suggestion @aduth . I think it is a good one and have updated my code.